surefire-providers/surefire-junit-platform/src/main/java/org/apache/maven/surefire/junitplatform/JUnitPlatformProvider.java

```diff
@@ -58,6 +58,11 @@
 import org.junit.platform.launcher.core.LauncherDiscoveryRequestBuilder;
 import org.junit.platform.launcher.core.LauncherFactory;
 
+/**
+ * JUnitPlatformProvider.
+ *
+ * @since 1.0
```

Comment:
Pls put here another version. We can make release the provider in e.g. 2.22.0.

Reply:
Fixing all JUnit-related version to `2.22.0`.